common/lib/common-utils/api-report/common-utils.alpha.api.md

```diff
@@ -4,6 +4,4 @@
 
 ```ts
 
-export { performance_2 as performance }
```

Comment:
A bit confused as to why it called it `performance_2` at any point... but the type is `@internal`.

Reply:
api-extractor uses that convention when it needs to rename things for whatever reason (usually two conflicting imports in the api file). It should be transparent to users. No context on visibility of the type.

Reply:
Yeah, I'm just confused what the second thing was. Maybe the native `performance` object, I guess.

Reply:
Yup. API-Extractor renames it to avoid conflicting with the global, then re-exports it under the original alias.